app/models/ext_management_system.rb

```diff
@@ -414,6 +416,14 @@ def self.ems_infra_discovery_types
     @ems_infra_discovery_types ||= %w(virtualcenter scvmm rhevm)
   end
 
+  def disable_ems
+    update_attributes(:enabled => false)
```

Comment:
I feel this is an important update which should a) run validations (which update_attributes does not) and record a changed date.

So 
```ruby
self.enabled = false
save!
```